lib/hets.rb

```diff
@@ -17,7 +17,7 @@ def self.minimum_version
 
   def self.qualified_loc_id_for(resource)
     locid = URI.escape(resource.versioned_locid)
-    "http://#{Settings.hostname}#{locid}"
+    "#{Hostname.url_authority}#{locid}"
```

Comment:
It is defined there, but the `url_authority` is a method and not an attribute of `config` ;)